drivers/watchdog/Kconfig

```diff
@@ -65,6 +65,15 @@ config WDT_COUNTER_CH_COUNT
 
 endif # WDT_COUNTER
 
+config WDT_EARLY_WARNING_INTERRUPT_SUPPORT
```

Comment:
This at end is a HW config, right? So, could it be defined in devicetree as a bool value ?